test/test_leet.py

```diff
@@ -0,0 +1,50 @@
+from test.conftest import MockUQCSBot, TEST_CHANNEL_ID
+import json
+from unittest.mock import patch
+
+LC_DIFFICULTY_MAP = ["easy", "medium", "hard"]
```

Comment:
Can't import this from the module?

Reply:
The DIFFICULTY_MAP? The other stuff I copied from the other test files.

Reply:
Yup

Reply:
This breaks the tests as !leet is not added officially yet.